ignite/cmd/doctor.go

```diff
@@ -0,0 +1,30 @@
+package ignitecmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/ignite/cli/ignite/services/doctor"
+)
+
+func NewDoctor() *cobra.Command {
+	return &cobra.Command{
+		Use:    "doctor",
+		Short:  "Fix chain configuration",
+		Hidden: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			session := cliui.New()
+			defer session.End()
+
+			doc := doctor.New(doctor.CollectEvents(session.EventBus()))
+
+			if err := doc.MigrateConfig(cmd.Context()); err != nil {
+				return err
+			}
+			if err := doc.FixDependencyTools(cmd.Context()); err != nil {
```

Comment:
```suggestion
return doc.FixDependencyTools(cmd.Context());
```

Reply:
The `doctor` command will be used to fix other kinds of thing, so more `if err := doc.FixXXX(); err != nil` will be added in the future, following `FixDependencyTools`. 

That's why I chose to keep the `if` statement.